libruncxx/memory.cc

```diff
@@ -141,6 +142,7 @@ namespace {
     cursor = cursor->next;
     cursor->next = 0;
     cursor->size = size - sizeof(MemChunk);
+    io::printf("[memory] Using region at %p, size %p\n", cursor, cursor->size);
```

Comment:
Let's not do that.

Reply:
ok. I still want to add a function to print out the memory layout used. It's the easiest way to get info about the memory layout of a device.